lib/organize.js

```diff
@@ -10,7 +10,14 @@ const { getLanguageService } = require('./get-language-service');
  */
 module.exports.organize = (
 	code,
-	{ filepath = 'file.ts', organizeImportsSkipDestructiveCodeActions, parentParser, parser },
+	{
+		filepath = 'file.ts',
+		organizeImportsSkipDestructiveCodeActions,
+		// @ts-ignore
+		organizeImportsMode = 'All',
```

Comment:
Do you really need to set a default value here, since we're already setting the default in the plugin options?

Also what is the `@ts-ignore` ignoring? 🙃

Reply:
Since the default value is unnecessary here, this one could be delete.